docs/content/manual/release-specific/v1.4.0/test-system-backup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Test Longhorn system backup
```

Comment:
This could be `Test Longhorn system backup syncing.`

Reply:
Reworded.